modules/nw-load-balancing-configure.adoc

```diff
@@ -0,0 +1,69 @@
+
+.Procedure
+
+== Define the default load balancer type
+
+When installing the cluster, you can specify the overall type of load balancer (Classic or NLB) that you want to use. This process defines a global setting that affects the default load balancer type that will be applied to the entire cluster if no specific type is defined elsewhere.
+
+[NOTE]
+====
+This example demonstrates how to define the default load-balancer type for a cluster deployed on AWS.
+====
+
+[source,yaml]
+----
+platform: <1>
+  aws: <2>
+    lbType: classic <3>
+----
+<1> The `platform` configuration key indicates the top-level key for platform-specific configuration settings in the OpenShift installation configuration file.
+<2> The `aws` configuration key specifies that the configuration settings under this key are specific to AWS as the infrastructure provider.
+<3> The `lbType` configuration key represents the load balancer type to be used for the cluster. Valid values are `classic` (classic load balancer) or `NLB` (network load balancer). If you do not specify a value, the `lbType` key defaults to `classic`.
+
+.Procedure
+
+== Specify load balancer behavior for an ingress controller
+
+After you install a cluster, you can configure your ingress controller to specify settings for how services (endpoints) are exposed to external networks, including specific parameters for the load balancer used by the ingress controller. This process allows for detailed control over the load balancer behavior, such as DNS management policy, connection idle timeout, and scope. These settings may override or complement the global platform configuration.
+
+[NOTE]
+====
+This example demonstrates how to define the default load-balancer type for a cluster deployed on AWS.
+====
+
+[source,yaml]
+----
+endpointPublishingStrategy: <1>
+  loadBalancer: <2>
+    dnsManagementPolicy: Managed <3>
+    providerParameters: <4>
+      aws: <5>
+        classicLoadBalancer: <6> 
+          connectionIdleTimeout: 0s <7>
+        type: Classic <8>
+      type: AWS <9>
+    scope: External <10>
+  type: LoadBalancerService <11>
+----
+<1> The `endpointPublishingStrategy' configuration key indicates the strategy used for exposing services (endpoints) to the external network.
+<2> The `loadBalancer' configuration key specifies the load balancer configuration settings.
+<3> The `dnsManagementPolicy` configuration key with a value of 'Managed' indicates that the record for the DNS management policy will be automatically managed.
+<4> The `providerParameters' configuration key contains specific parameters for the load balancer provider.
+<5> The `aws` parameter specifies that the load balancer provider is AWS.
+<6> The `classicLoadBalancer` key contains settings specific to the CLB on AWS.
+<7> The `connectionIdleTimeout' key with a value of `0s` sets the connection idle timeout for the CCLBto 0 seconds. This value can be adjusted based on the desired idle timeout for connections.
```

Comment:
The `connectionIdleTimeout' key with a value of `0s`(or nil) sets the connection idle timeout for the CLB to the default 60 seconds. This value can be adjusted based on the desired idle timeout for connections.